src/go/k8s/config/manager/manager.yaml

```diff
@@ -29,7 +29,7 @@ spec:
         - /manager
         args:
         - --leader-elect
-        image: vectorized/redpanda-operator:2021022-e853bf3
+        image: vectorized/redpanda-operator:2021022-e853bf3 
```

Comment:
NIT: unrelated changes. 

P.S. If you install pre-commit there is one check that will remove trailing spaces.

Reply:
oh yeah it's just the newline

okay, will do that